lib/vssh/libssh.c

```diff
@@ -891,12 +900,14 @@ static CURLcode myssh_statemach_act(struct Curl_easy *data, bool *block)
         sshc->authed = TRUE;
         infof(data, "completed keyboard interactive authentication");
       }
+      else {
+        MOVE_TO_PASSWD_AUTH;
```

Comment:
Seems missing a break; after this line, or should move following AUTH DONE line up to if block.

Reply:
Good catch, thanks!